frontend/lib/src/components/widgets/DataFrame/arrowUtils.ts

```diff
@@ -436,8 +438,8 @@ export function getCellFromArrow(
 
   if (!column.isEditable) {
     // Only apply display content and css styles to non-editable cells.
-    if (notNullOrUndefined(arrowCell.displayContent)) {
-      const displayData = removeLineBreaks(arrowCell.displayContent)
+    if (styledCell && notNullOrUndefined(styledCell?.displayContent)) {
```

Comment:
nit: wouldn't `if (notNullOrUndefined(styledCell?.displayContent))` work too because if `styledCell` is `undefined`, `undefined` will be passed to `notNullOrUndefined` and also be true? Or does the linter complain?

Reply:
> Or does the linter complain?

 yeah, somehow there was a linter complain about `styledCell` being undefined... maybe the linter is not smart enough for this situation?

Reply:
Oh ok 👍